e2e/cypress/integration/messaging/typing_on_middle_spec.js

```diff
@@ -21,9 +21,9 @@ describe('Messaging', () => {
         cy.get('#post_textbox').clear().type('aa');
 
         // # Move the cursor and keep typing
-        cy.get('#post_textbox').click().type('{leftarrow}b');
+        cy.get('#post_textbox').click().type('{leftarrow}bc');
```

Comment:
I think you might need to wait a moment before typing the `c` -- Just a hunch, but that would give it time to move the cursor (if it's going to). Typing these characters too rapidly might not trigger the problem we're trying to test.